invenio_circulation/records/loaders/schemas/json.py

```diff
@@ -92,7 +95,7 @@ def get_pid_field(self):
     request_expire_date = DateString()
     request_start_date = DateString()
     start_date = DateString()
-    transaction_date = DateTimeString()
+    transaction_date = DateTimeString(missing=transaction_date_default)
```

Comment:
```suggestion
    transaction_date = DateTimeString(missing=arrow.utcnow())
```
can be a value or callable